extra/2-piping.js

```diff
@@ -16,26 +16,32 @@
   the final result to the variable goodCode
 */
 
-function add() {
-
+function add(num1, num2) {
+  return num1 + num2;
 }
 
-function multiply() {
+function multiply(num1, num2) {
+  return num1 * num2;
 
 }
 
-function format() {
+function format(num1) {
+  return `£${num1}`
 
 }
 
 const startingValue = 2;
 
 // Why can this code be seen as bad practice? Comment your answer.
-let badCode =
+
+let badCode = format (multiply(add(startingValue, 10),2))
+
+
 
 /* BETTER PRACTICE */
 
-let goodCode =
+let goodCode = format(multiply(add(startingValue, 10),2));
```

Comment:
Here the idea is that using too many functions in one line can become hard to read. It is better to try to save the result of each computation into a new variable and then use that variable in the next function call. How could you do that?